doc/akri_connector/connector-metadata-schema.json

```
@@ -467,8 +442,18 @@
                             "alternativeTypeName" : { 
                               "$ref" : "#/definitions/alternativeTypeName",
                               "description": "The alternative name to the type \"dataPoints\" within an event"
+                            },
+                            "fields": {
+                              "type": "object",
+                              "description" : "Describes if this connector expects these fields to be provided in the asset defintion when deploying the connector. Also describes the expected shape of these values if they are supported.",
```

```suggestion
                              "description" : "Describes if this connector expects these fields to be provided in the data point definition when deploying the connector. Also describes the expected shape of these values if they are supported.",
```

Is this specific for data points?

---

Good idea. This was copy-pasted, but I can make these more specific